gateway/src/apicast/configuration.lua

```diff
@@ -140,21 +142,47 @@ function _M.services_limit()
 end
 
 function _M.filter_services(services, subset)
-  subset = subset and util.to_hash(subset) or _M.services_limit()
-  if not subset or not next(subset) then return services end
+  local selected_services = {}
+  local service_regexp_filter  = env.value("APICAST_SERVICES_FILTER_BY_URL")
```

Comment:
`env.value` will never return empty string, so you can just use `or default_regexp`.
https://github.com/3scale/lua-resty-env/blob/41ea2d44bd1b80445e7f1e5fa7dfa1f9a7cd47c7/src/resty/env.lua#L106-L122